rtrlib/rtr/packets.c

```diff
@@ -20,6 +20,8 @@
 #define TEMPORARY_PDU_STORE_INCREMENT_VALUE 100
 #define MAX_SUPPORTED_PDU_TYPE 10
 
+static int interval_option = DEFAULT_MIN_MAX;
```

Comment:
This is only visible here, but you also define it in `rtr.c`, hence it should be globally defined (and set) somewhere